packages/console/src/pages/NotFound/index.tsx

```diff
@@ -1,7 +1,20 @@
 import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import errorIcon from '@/assets/images/error-icon.svg';
```

Comment:
nit: an image like this can not be called as an "icon"?

Reply:
Yes, I would like to call it something like `errorImage`, but the file name is `error-icon.svg`, which is copied from ui part...
If I rename it, would it cause any confusion? Or maybe I should change the name in both ac and ui

Reply:
Updated